docs/rename.md

```diff
@@ -0,0 +1,166 @@
+# Git Credential Manager Rename
+
+In November 2021, _"Git Credential Manager Core"_ was [renamed][rename-pr] to
+simply _"Git Credential Manager"_, dropping the "Core" moniker. We announced the
+new name in a [GitHub blog post][rename-blog], along with the new home for the
+project in it's own [organization][gcm-org].
+
+![Git Credential Manager Core renamed](img/gcmcore-rename.png)
+
+At the time, the actual exectuable name was not updated and continued to be
+`git-credential-manager-core`. As of [VERSION][rename-ver], the executable has
+been renamed to `git-credential-manager`, matching the new project name.
+
+## Rename transition
+
+If you continue to use the `git-credential-manager-core` executable name you may
+see warning messages like below:
+
+```console
+warning: git-credential-manager-core was renamed to git-credential-manager
+warning: see https://aka.ms/gcm/rename for more information
+```
+
+Since the executable was renamed in VERSION, GCM has also included symlinks
+using the old name in order to ensure no one's setups would immediately break.
+
+These links will remain until _two_ major Git versions are released after GCM
+VERSION, _**at which point the symlinks will no longer be included**_.
```

Comment:
I think giving two major Git versions is a good runway. Thoughts? @ldennington @dscho

(My "major" of course I mean `y` in `x.y.z`.. not `x`!)

Reply:
Sure!